luda-editor/luda-editor-client/src/app/editor/clip_editor/mod.rs

```diff
@@ -37,9 +37,9 @@ impl ClipEditor {
 
     pub fn render(&self, props: &ClipEditorProps) -> RenderingTree {
         match &self {
-            ClipEditor::Camera(camera_clip_editor) => match props.clip {
+            ClipEditor::Camera(camera_clip_editor) => match &props.clip {
                 Clip::Camera(camera_clip) => camera_clip_editor.render(&CameraClipEditorProps {
-                    camera_clip,
+                    camera_clip: &camera_clip,
```

Comment:
Don't care about this file, I make type `Clip` use Arc instead of &.

Reply:
This -> https://github.com/NamseEnt/namseent/pull/96/files#diff-c003fe01a9671c25aabe0624cc90fa06d368150857c69ab320f73bcd5b8b26b8R109